SCClassLibrary/Common/Collections/Collection.sc

```diff
@@ -112,12 +112,18 @@ Collection {
 	remove { ^this.subclassResponsibility(thisMethod) }
 	removeAll { | list | list.do { | item | this.remove(item) } }
 	removeEvery { | list | this.removeAllSuchThat(list.includes(_)) }
+
+	removeAllEqual { | list |
+		list.do { | removing |
+			var index = this.detectIndex({ |item| item == removing });
```

Comment:
this will only remove the first match of `removing` in this collection.

`[5,5,5,3,3].removeAllEqual([5,3])` -> `[5, 5, 3]`.

Reply:
ok, I'll change the name to `removeEachEqual`.

Reply:
what is the intended behavior though? i don't understand how this case is general enough to warrant a separate method

Reply:
i didn't realize that was the intended behavior. isn't this just an equality-based version of `difference` now?

Reply:
in fact, that would probably be better, because `removeAt` is not part of the interface of `Collection`.

Reply:
yes. `difference` is implemented by `removeAll` and `copy`. So I modeled `removeAllEqual` on `removeAll`. In that way, removeAllEqual was a good name, even if misleading perhaps.

It is intended, because when you merge lists, you don't want to remove all that are equal but only each.